### DISCUSSION TITLE
Fix "snapback" effect; prevent horizontal wheel scrolling

### DISCUSSION THREAD

**Discussion by ElenaXX** [ramensoftware/windhawk#364] [Ideas]
These would be really appreciated, other than there is no tool or patch at present to fix these annoying Windows "bug-features":
1. Prevent scrollbars "snapback" effect, which affects all Windows versions and is *frustrating*
2. Prevent horizontal scrolling when moving the mouse wheel on a window content which can be just scrolled horizontally

PS. I posted this already but for some strange reason my post was removed.

**Comment by m417z** [comment]
Your post is here: https://github.com/ramensoftware/windhawk-mods/discussions/1379, which makes sense as it's about Windhawk mods, not about Windhawk itself.

**Reply by ElenaXX** [reply]
Oh very very sorry ! Lol that's why :) I simply ended up in the wrong section ;) I apologize, I am not very skilled with github !
